docs/quickstart.md

```diff
@@ -0,0 +1,411 @@
+
+
+petstore.yaml: validated in 72ms
+
+❌ Validation failed with 3 errors and 1 warning.
+run with `--generate-ignore-file` to add all problems to ignore file.
+```
+
+Yikes! There are 3 errors!
+
+Don't worry, they were made intentionally to show the power of the tool.
+
+Let's fix the errors. Please, make the following modifications:
+
+```bash line 15
+operationIds: --> operationId:
+```
+
+```bash line 58
+/pets/{petId: --> /pets/{petId}:
+```
+
+```bash line 103
+require: --> required:
+```
+
+Once fixed, run the `lint` command again:
+
+```bash
+$ openapi lint petstore.yaml
+```
+
+You should get the following response:
+
+```bash
+$ openapi lint petstore.yaml 
+No configurations were defined in extends -- using built in recommended configuration by default.
+
+validating petstore.yaml...
+petstore.yaml: validated in 44ms
+
+Woohoo! Your OpenAPI definition is valid. 🎉
+```
+
+Congratulations! You did a great job and now your OpenAPI definition is valid!
+
+### split
+
+With this command, your OpenAPI definition file will be divided into constituent parts to follow the multi-file approach. `split` does the opposite of the [`bundle`](#bundle) command.
+
+In your terminal, type the following:
+
+```bash
+$ openapi split petstore.yaml --outDir openapi
+```
+
+You should get the following response:
+
+```bash
+$ openapi split petstore.yaml --outDir openapi
+Document: petstore.yaml is successfully split
+ and all related files are saved to the directory: openapi 
+
+petstore.yaml: split processed in 44ms
+```
+
+Cool, split completed. But what actually happened?
+
+Open the `openapi` directory. See? That's where the magic took place:
+
+```bash
+.
+├── components
+│   └── schemas
+│       ├── Error.yaml
+│       ├── Pet.yaml
+│       └── Pets.yaml
+├── paths
+│   ├── pets.yaml
+│   └── pets@{petId}.yaml
+└── openapi.yaml
+```
+
+With the `split` command, you can easily switch to a multi-file structure in a blink of an eye even if your definition file became extremely lengthy.
+
+### bundle
+
+With this command, you can merge standalone files back into a single definition file. `bundle` does the opposite of the [`split`](#split) command.
+
+Let's bundle the definition file that we split earlier.
+
+In your terminal, type the following:
+
+```bash
+$ openapi bundle openapi/openapi.yaml --output bundled.yaml
+```
+
+You should get the following response:
+
+```bash
+$ openapi bundle openapi/openapi.yaml --output bundled.yaml
+bundling openapi/openapi.yaml...
+📦 Created a bundle for openapi/openapi.yaml at bundled.yaml 46ms.
+```
+
+You should now see a new `bundled.yaml` file as a result of the `bundle` command completion. Open this file in your text editor or IDE and compare it with the linted definition file. You can see that both files are the same which means that the `bundle` command executed flawlessly.
+
+### preview-docs
+
+With this command, you can run a server to preview your OpenAPI definition file in real-time mode.
```

Comment:
This is technically what happens in the background, but not quite what most users would expect or want to do from a command that's called preview-docs :) Could you try rephrasing this so it involves the fact that this generates a preview of the API documentation?